src/globals.rs

```diff
@@ -0,0 +1,131 @@
+
+use holochain_wasm_utils::SinglePageStack;
+use init_globals::init_globals;
+use init_globals::InitGlobalsOutput;
+
+pub type HashString = String;
+
+//--------------------------------------------------------------------------------------------------
+// ZOME APP GLOBALS
+//--------------------------------------------------------------------------------------------------
+
+lazy_static! {
+  pub static ref APP_GLOBALS: InitGlobalsOutput = init_globals();
+
+  pub static ref APP_NAME: String               = APP_GLOBALS.clone().app_name;
+  pub static ref APP_DNA_HASH: HashString       = APP_GLOBALS.clone().app_dna_hash;
+  pub static ref APP_KEY_HASH: HashString       = APP_GLOBALS.clone().app_key_hash;
+  pub static ref APP_AGENT_HASH: HashString     = APP_GLOBALS.clone().app_agent_hash;
+  pub static ref APP_AGENT_TOP_HASH: HashString = APP_GLOBALS.clone().app_agent_top_hash;
+  pub static ref APP_AGENT_STR: String          = APP_GLOBALS.clone().app_agent_str;
+}
+
+// Internal global for memory usage
+pub static mut g_mem_stack : Option<SinglePageStack> = None;
```

Comment:
This could just as well be a `const`

Reply:
We can't mutate a `const` so I am not sure how that would work?